xml/admin_caasp_cluster.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "generic-entities.ent">
+    %entities;
+]>
+<chapter xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0"
+  xml:id="admin-caasp-cluster">
+ <!-- ============================================================== -->
+ <title>&ceph; cluster administration</title>
+ <info>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:translation>yes</dm:translation>
+   <dm:release>SES 7</dm:release>
+  </dm:docmanager>
+ </info>
+ <para>
+  This chapter introduces tasks that are performed on the whole cluster.
+ </para>
+ <sect1 xml:id="admin-caasp-cluster-shutdown">
+  <title>Shutting down and restarting the whole cluster</title>
+
+  <para>
+   To shut down the whole &ceph; cluster for planned maintenance tasks, follow
+   these steps:
+  </para>
+
+  <procedure>
+   <step>
+    <para>
+     Stop all clients that are using the cluster.
```

Comment:
@shaas could you provide the commands here? Thank you

Reply:
Well, that depends on the client which relies on the storage. I am not sure what commands you are expecting.